deploy/charts/external-secrets/values.yaml

```diff
@@ -48,6 +48,9 @@ createOperator: true
 # a time.
 concurrent: 1
 
+# -- If set External secret will reuse the AWS session without creating a new one on each request.
+enableAWSSession: false
```

Comment:
Users can set `extraArgs: ["--experimental-enable-aws-session-cache"]`, i would like to avoid adding it in the values.yaml and the extra `if` branch in the deployment.yaml above. (helm values is _sort of_ part of our deprecation policy)